doc/modules/changes/20240625_gassmoeller

```diff
@@ -0,0 +1,5 @@
+Removed: The input parameter 'Update ghost particles' has been removed.
```

Comment:
It is not really removed anymore, right?

Reply:
True, I changed it to deprecated.